src/commons/big_query/big_query_job_reference.py

```diff
@@ -19,3 +24,14 @@ def __eq__(self, other):
 
     def __ne__(self, other):
         return not (self == other)
+
+    def create_post_copy_action(self, copy_job_request):
```

Comment:
As discussed, does this method really belong here? BigQueryJobReference looks bit like POJO.